packages/common/client-utils/src/trace.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { performance } from "./indexNode.js";
+import { performance } from "./performanceIsomorphic.js";
```

Comment:
This is the actual fix.  There is no node/browser specific code involved.  PerformanceIsomorphic just narrows the type of the global performance object to the intersection of what browser/node support.